### DISCUSSION TITLE
Rescalable axis accuracy graph

### DISCUSSION THREAD

**Discussion by AbelHutten** [monkeytypegame/monkeytype#4762] [Ideas] [answered]
The y-axis on the personal statistics graph goes from 0% to 100%, and there is no way to rescale this. This makes it hard to see if there is a trend in accuracy, since it tends to only be in a very small sub range. In the image below, for example, I have a hard time to see the progression of my accuracy. If I could choose to set a different y-axis range, that could be very helpful in my opinion.

![Screenshot from 2023-11-04 17-28-26](https://github.com/monkeytypegame/monkeytype/assets/122212042/1c1f618b-55b1-4053-b010-13b11e03c1f8)

**Comment by Miodec** [comment]
The scale is setup like this to make sure the data points dont overlap. You can still see the trend with the average of 10 and average of 100 lines.

**Reply by AbelHutten** [reply]
I get that, but I would like to have the option to give accuracy center stage, instead of always having it as a small sliver at the bottom.

**Reply by Miodec** [reply]
How about a 4th button which toggles results on and off. If results are hidden, the accuracy is scaled to fit the whole height of the graph.

**Reply by AbelHutten** [reply]
Yes that sounds great!

**Reply by Miodec** [reply, marked as answer]
Alright, will be live later today.

**Reply by AbelHutten** [reply]
It's great, thank you!
